src/Framework/Collection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Give\Framework;
```

Comment:
Let's start with the interface inside of the Domain - we can extract it later as needed.

Reply:
@kjohnson You are correct.